LCJava/src/main/java/tw/TrainMapException.java

```diff
@@ -0,0 +1,16 @@
+package tw;
+
+public class TrainMapException extends Exception{
+    String errorMsg;
+
+    public TrainMapException() {
+        this.errorMsg = "No error";
```

Comment:
is there any case when we want to throw an exception with "no error" as its error message?